tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -32,64 +32,14 @@ parameters:
   type: string
   default: none
 
-- name: STORAGE_ACCOUNT
-  type: string
-  default: none
-
-- name: STORAGE_KEY
-  type: string
-  default: none
-
 # Set version
 steps:
 
-# These steps should ONLY run if we're using the repo version of the build tools. These steps are mutually exclusive
-# with the next group of steps.
-- ${{ if eq(parameters.buildToolsVersionToInstall, 'repo') }}:
-  - template: include-install-pnpm.yml
-    parameters:
-      buildDirectory: $(Build.SourcesDirectory)/build-tools
-      pnpmStorePath: ${{ parameters.pnpmStorePath }}
-      enableCache: false
-
-  - task: Bash@3
-    name: InstallBuildTools
-    displayName: Install Fluid Build Tools (from repo)
-    inputs:
-      targetType: 'inline'
-      workingDirectory: $(Build.SourcesDirectory)/build-tools
-      script: |
-        pnpm i --frozen-lockfile
-        pnpm build:compile
-        cd packages/build-cli
-        # Use npm link instead of pnpm link because it handles bins better
-        npm link
-        echo "which flub: $(which flub)"
-
-# These steps install a version of build-tools from the npm registry. As noted above, these steps are mutually exclusive
-# with the previous group of steps.
-- ${{ if ne(parameters.buildToolsVersionToInstall, 'repo') }}:
-  - task: Bash@3
-    name: InstallBuildTools
-    displayName: Install Fluid Build Tools (from npm)
-    inputs:
-      targetType: 'inline'
-      workingDirectory: ${{ parameters.buildDirectory }}
-      script: |
-        echo "${{ parameters.buildToolsVersionToInstall }}"
-        npm install --global "@fluid-tools/build-cli@${{ parameters.buildToolsVersionToInstall }}"
-
-- task: Bash@3
-  name: BuildToolsInstallCheck
-  displayName: Check Build Tools Installation
-  inputs:
-    targetType: 'inline'
-    workingDirectory: ${{ parameters.buildDirectory }}
-    script: |
-      # Output the help and full command list for debugging purposes
-      echo "which flub: $(which flub)"
-      flub --help
-      flub commands
+- template: include-install-build-tools.yml
```

Comment:
tested this change: https://dev.azure.com/fluidframework/internal/_build/results?buildId=281503&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=5cfe65a4-d7cb-5934-e953-1c67617821f4